src/schemas/ERC1155/index.ts

```diff
@@ -1,12 +1,12 @@
 import * as Web3 from 'web3';
-
 import {
   FunctionInputKind,
   FunctionOutputKind,
   Schema,
-  StateMutability,
+  StateMutability
```

Comment:
nit: can we add an eslint rule for this? my editor keeps going back and forth and is very confused

Reply:
you using prettier?

Reply:
Just going to add the comma back, don't think this repo has normal eslint setup so it's going to be messy until we actually add that in, but out of scope